src/api/route-services/experiment.js

```diff
@@ -116,7 +124,9 @@ class ExperimentService {
     };
 
     const result = await dynamodb.updateItem(params).promise();
-    return result;
+    const prettyData = convertToJsObject(result.Attributes);
+
+    return prettyData;
```

Comment:
Return the pretty data instead of the weird DynamoDB convention when we do a PUT, as the REST conventions dictate.